components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -203,6 +203,21 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 @property(nonatomic) UIImageRenderingMode imageRenderingMode;
 
+/**
+ Determines if a divider should be shown between the header and actions. To customize the divider
+ color see @c headerDividerColor.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL showHeaderDivider;
```

Comment:
Nit/naming: I believe `showsHeaderDivider` or `headerDividerVisible` would better align with Apple's naming conventions.
*  `setShowsHeaderDivider` (setter) and `showsHeaderDivider` (getter)
*  `setHeaderDividerVisible` (setter) and `isHeaderDividerVisible` (getter)

From [Apple's Naming guidelines](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html), "If the property is expressed as a verb, …the verb should be in the simple present tense."

Reply:
This has been updated.